gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/CloudEventHandlerConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.config;
+
+import io.cloudevents.spring.mvc.CloudEventHttpMessageConverter;
+import java.util.List;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class CloudEventHandlerConfiguration implements WebMvcConfigurer {
+
+  @Override
+  public void configureMessageConverters(List<HttpMessageConverter<?>> converters) {
+    converters.add(0, new CloudEventHttpMessageConverter());
```

Comment:
Doing this twice seems redundant?

Reply:
As we have different endpoints written in gate-core and gate-web separately, we need this configuration to handle the CloudEvent conversion by Spring framework.

Reply:
I'm pretty sure that beans defined in core also show up in web regardless if it's an api or implementation dependency.

Reply:
yes apologies, it works keeping this config file in either core or web. I will delete this config file from web